src/schema/v2/me/collection.ts

```diff
@@ -16,26 +24,74 @@ export const Collection: GraphQLFieldConfig<any, ResolverContext> = {
     description: "A collection of artworks",
     fields: () => ({
       ...InternalIDFields,
-      name: {
-        type: new GraphQLNonNull(GraphQLString),
-        description:
-          "Name of the collection. Has a predictable value for 'standard' collections such as Saved Artwork, My Collection, etc. Can be provided by user otherwise.",
+      artworksConnection: {
+        type: artworkConnection.connectionType,
+        args: {
+          ...pageable(),
```

Comment:
works but you can also do `pageable({sort: ...})` (ie- you can include all the additional args that way vs. spreading it yourself here) #minor